### PR TITLE
Base admin interface

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

File: db/migrate/20160705085336_rename_template_model.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class RenameTemplateModel < ActiveRecord::Migration[5.0]
   def change
-    rename_table :templates, :site_templates
+    rename_table :site_templates, :site_templates
```
Comment (adammulligan) [inline db/migrate/20160705085336_rename_template_model.rb]: I think your find and replace might have gone awry here, as renaming a table to the same name makes Postgres unhappy 😀

**Comment by adammulligan** [comment]
Looks good. As `rake db:sample` isn't standard, can you also update the README to let people know that it might be useful to run it first in development? Thanks!

**Comment by tiagojsag** [comment]
@adammulligan anything else?

**Comment by adammulligan** [comment]
@tiagojsag yeah, just the other comment I made about the db migration as it causes an exception to be thrown. Happy to merge once that's fixed :+1:

**Comment by tiagojsag** [comment]
oh missed that comment. sry

**Pull request merged**